Add documentation links to all main page headings WD-6525

edlerd opened the pull request:
## Done

- added an info icon linking to documentation for most main headings

Fixes WD-6525

## QA

1. Run the LXD-UI:
    - On the demo server via the link posted by @webteam-app below. This is only available for PRs created by collaborators of the repo. Ask @lorumic or @edlerd for access.
    - With a local copy of this branch, run as described [here](../HACKING.md#setting-up-for-development).
2. Perform the following QA steps:
    - check the headings of all pages from the main navigation. Most should have an info icon linking to the docs of that page.

webteam-app commented:
Demo starting at https://lxd-ui-483.demos.haus

piperdeck reviewed (changes requested):
I think this is a pretty safe change, but I'd make 2 minor changes:

1. I would change the icon to the "i" icon so that it matches the documentation icon in the side navigation
2. I would align the lower edge of the icon with the baseline of the header

Other than that, LGTM!

edlerd commented:
> I think this is a pretty safe change, but I'd make 2 minor changes:
> 
> 1. I would change the icon to the "i" icon so that it matches the documentation icon in the side navigation
> 2. I would align the lower edge of the icon with the baseline of the header

For 1. I changed the icon.
For 2. The icon was baseline aligned. I think because of the circle outline, it seemed to sit a bit too high from the baseline. I added a bit of top margin (which changes on very large screen widths, so better test it responsively with various widths). Wdyt?

piperdeck reviewed (approved):
Thanks for the additional info,

I think it probably also has to do with the fact that the Ubuntu Font is designed in such a way the the round bottoms of letters hang slightly below the baseline. I wouldn't worry about it too much, I think this is good to go.

lorumic commented on src/pages/instances/InstanceList.tsx:
```diff
@@ -478,7 +479,14 @@ const InstanceList: FC = () => {
+            <h1 className="p-heading--4 u-no-margin--bottom">
+              <HelpLink
+                href="https://documentation.ubuntu.com/lxd/en/latest/explanation/instances/#expl-instances"
+                title="Learn more about Instances"
```
```suggestion
                title="Learn more about instances"
```

lorumic reviewed (approved):
It's good, just one tiny suggestion.
